### PR TITLE
feat: bot.respawn, fix respawn with flying squid

### PR DESCRIPTION
**Pull request by zardoy**
respawn wasnt working with flying squid. It worked with vanilla client as I understand vanilla handle both payload and actionId. Maybe it would be better to remove acknowledgment of respawnIsPayload feature?

### CODE REVIEW

**Comment by zardoy** [comment]
imo `respawn` option should be renamed to `autoRespawn`

**Comment by zardoy** [comment]
so confused why it was closed

File: lib/plugins/health.js [hunk 1]
```diff
@@ -25,10 +25,17 @@ function inject (bot, options) {
...
       bot.emit('spawn')
     }
   })
+
+  const respawn = () => {
+    if (bot.isAlive) return
+    bot._client.write('client_command', bot.supportFeature('respawnIsPayload') ? { payload: 0 } : { actionId: 0 })
+  }
+
+  bot.respawn = respawn
```
Comment (rom1504) [inline lib/plugins/health.js:40]: this should be added to API.md

Comment (zardoy) [reply lib/plugins/health.js:40]: done, thanks

**Pull request merged**